cfgov/jinja2/v1/_includes/molecules/info-unit.html

```diff
@@ -49,8 +49,13 @@
          change to a boolean parameter. #}
 <div class="m-info-unit
             {{ 'm-info-unit__' ~ value.modifier if value.modifier else '' }}">
-
     {% if value.image %}
+        {# TODO: This code was added for the sake of A / B testing. #}
+        {% set use_image_link = request.url.endswith( '/about-us/careers' )
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2554/commits/ac7daa93d602e820edb43584a948b9d28f7b8839